server/chat.ts

```diff
@@ -984,7 +984,7 @@ export class CommandContext extends MessageContext {
 				}
 				if (targetUser.settings.blockPMs &&
 					(targetUser.settings.blockPMs === true || !Users.globalAuth.atLeast(user, targetUser.settings.blockPMs)) &&
-					!user.can('lock')) {
+					!user.can('lock') && toID(targetUser) != toID(user)) {
```

Comment:
```suggestion
					!user.can('lock') && targetUser.id != user.id) {
```

Reply:
(if you use toID on a user object, it returns user.id, so it's just cleaner this way)

Reply:
This should also be `!==` not `!=`.